static/local/blackwomenssuffrage/key-figures.md

```diff
@@ -1,83 +1,199 @@
-* https://www.battlefields.org/learn/primary-sources/fugitive-slave-act
+## Angelina Weld Grimk&eacute;
+### 1880&#8211;1958
+Angelina Weld Grimk&eacute; was born into a prominent biracial abolitionist family from Boston. Her father was Archibald Grimk&eacute;, the son of a white plantation owner and enslaver and an enslaved woman of mixed race, and the nephew of the well-known white abolitionist and suffragist sisters Sarah and Angelina Grimk&eacute;. Her mother Sarah Stanley was from a middle-class Midwestern family. Grimk&eacute;'s parents split when she was just 3 years old, and although she initially lived with white mother, at 7 she was returned to her father's care and never saw her mother again. Raised by her well-off father, who was a lawyer and the executive director of the [National Association for the Advancement of Colored People](/search?q=National+Association+for+the+Advancement+of+Colored+People) (NAACP), Grimk&eacute; was educated at some of the best schools in Massachusetts. She became an English teacher in her early 20s, working at the Armstrong Manual Training School and what would become Dunbar High School, both in Washington, D.C., and continuing her education in the summers. At Dunbar, Grimk&eacute; made the acquaintance of a group of Black women writers, and began writing about her observations of black life and racism, among other topics. Her writing, which included poetry, plays, essays and short stories, is often described as having helped lay the groundwork for the Harlem Renaissance. Among Grimk&eacute;'s most well-known works is the play Rachel, published in 1920 in response to NAACP founder [W.E.B. Dubois's]() call for the creation of theater by and for Black people. The play tells the story of a young Black woman who decides she will not bring children into a world plagued by racism. According to historian Elizabeth Brown-Guillory, it was the first play written by a Black woman to become a "fully staged professional production." In addition, Grimk&eacute;'s poetry has been included in several prominent anthologies of Black poetry. A supporter of women's suffrage like her well-known aunts, Grimk&eacute; believed that winning the vote was essential to ending injustice against women. She developed friendships with prominent suffragists and activists [Mary Church Terrell](/search?q=Mary+Church+Terrell) and [Coralie Franklin Cook](/search?q=Coralie+Franklin+Cook), both of whom were involved in the [National American Woman Suffrage Association](/search?q=National+American+Woman+Suffrage+Association) and black women's club movement, and she was active in an influential group of Washington, D.C.-based Black women suffragists that included Terrell and Cook, as well as [Nannie Helen Burroughs](/search?q=Nannie+Helen+Burroughs), [Anna Julia Cooper](/search?q=Anna+Julia+Cooper) and others. Grimk&eacute; died in 1958 in New York City. Her work is considered to have helped set the stage for the Harlem Renaissance.
```

Comment:
You can add a link Angela Weld Grimke `/search?q=Angelina+Weld+Grimk&eacute;` (this time you do need the e with accent in the link)
Change `/search?q=Anna+Julia+Cooper` to `/search?q=Anna+Cooper`
Change `/search?q=Nannie+Helen+Burroughs` to `/search?q=Nannie+Burroughs`